docs/manual/scala/releases/Migration16.md

```diff
@@ -19,3 +19,9 @@ addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.6.0")
 ```
 
 We also recommend upgrading to sbt 1.2.8 or later, by updating the `sbt.version` in `project/build.properties`.
+
+## Minor changes
+
+### JSON Compression threshold
+
+When creating a serializer with `JsonSerializer.compressed[T]` compression will only kick in when the serialized representation is biger than a threshold. The default value for `lagom.serialization.json.compress-larger-than` has been increased from 1024 bytes to 32 Kilobytes. (See [#1983](https://github.com/lagom/lagom/pull/1983))
```

Comment:
there is one mention of default value 1024 in Serialization.md too

Reply:
thanks, fixing!